app/merginapistatus.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MERGINAPISTATUS_H
+#define MERGINAPISTATUS_H
+#pragma once
+
+#include <QObject>
+
+class MerginApiStatus
+{
+    Q_GADGET
+  public:
+    explicit MerginApiStatus();
+
+    enum VersionStatus
+    {
+      UNKNOWN, // unchecked
+      NOT_FOUND, // cannot be checked due to network/wrong url
+      PASSED,
+      FAILED
```

Comment:
please document what does this mean (supposedly the version test has failed). Maybe also better to rename to something like "incompatible version" as there are more ways to fail? (e.g. "not found" is also a kind of failure)